lib/functions.sh

```diff
@@ -81,18 +95,23 @@ buildx() {
     OPTIONAL_TARGET="--target $target"
   fi
 
-  docker buildx build \
-    -f $file \
-    --build-arg BUILDKIT_INLINE_CACHE=1 \
-    --build-arg CI_BRANCH \
-    --build-arg CI_STRING_TIME \
-    --cache-from $BK_ECR:$APP-$tag-cache-$cache_id \
-    --cache-from $BK_ECR:$APP-$tag-cache-master \
-    --secret id=railslts,env=BUNDLE_GEMS__RAILSLTS__COM \
-    --secret id=jfrog,env=BUNDLE_SAGEONEGEMS__JFROG__IO \
-    --ssh default $OPTIONAL_TARGET \
-    -t $REPO:$tag \
-    .
+  args=(
+    "-f $file"
+    "--build-arg BUILDKIT_INLINE_CACHE=1"
+    "--build-arg CI_BRANCH"
+    "--build-arg CI_STRING_TIME"
+    "--cache-from $BK_ECR:$APP-$tag-cache-$cache_id"
+    "--cache-from $BK_ECR:$APP-$tag-cache-master"
+    "--secret id=railslts,env=BUNDLE_GEMS__RAILSLTS__COM"
+    "--secret id=jfrog,env=BUNDLE_SAGEONEGEMS__JFROG__IO"
+    "--ssh default $OPTIONAL_TARGET"
+    "-t $REPO:$tag"
+    "${build_args/"--build_arg"/"--build-arg"}"
```

Comment:
Does this replacement still work if there are multiple build arg? This test suggests not..
```bash
$ var='foo bar wibble bar'
$ echo ${var/bar/meep}
foo meep wibble bar
```